### PR TITLE
Update QUEST_LV_0100.tsv

### PR DESCRIPTION
**Pull request by seejy**
Minor proof-reading from lines #9300-9600

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -9510,48 +9510,48 @@ QUEST_LV_0100_20151001_009505	You are being lured to some mysterious power.
...
+QUEST_LV_0100_20151001_009517	The elder told you that it was after the demons attacked the village. The elder told you that the demonic totem which was set by the demons brainwashed the Pharets and asked you to destroy the demonic totem. 
 QUEST_LV_0100_20151001_009518	You've defeated the demons that were protecting the demonic totem and destroyed the totem. Return to the elder and tell him about it.
-QUEST_LV_0100_20151001_009519	The elder thanked you for breaking the demonic totem. Listen from the elder what you will do from now on.
-QUEST_LV_0100_20151001_009520	Look for the traces of the mysterious girl
-QUEST_LV_0100_20151001_009521	The elder of the village told you that while you've gone to destroy the demoninc totem, the girl appeared at the village. He said the girl disappeared to the gigantic statue of the goddess without any words. The elder said if you are the outsider from the folklore, you would be able to find the traces of the girl.
+QUEST_LV_0100_20151001_009519	The elder thanked you for breaking the demonic totem. Listen to the elder on what you will do from now on.
+QUEST_LV_0100_20151001_009520	Look for traces of the mysterious girl
```
Comment (zhouyu47) [inline QUEST_LV_0100.tsv]: demonic

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -9510,48 +9510,48 @@ QUEST_LV_0100_20151001_009505	You are being lured to some mysterious power.
...
+QUEST_LV_0100_20151001_009521	The elder of the village told you that while you've gone to destroy the demonnic totem, the girl appeared at the village. He said the girl disappeared to the gigantic statue of the goddess without a word. The elder said if you are the outsider from the folklore, you would be able to find traces of the girl.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: demonic

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -9510,48 +9510,48 @@ QUEST_LV_0100_20151001_009505	You are being lured to some mysterious power.
...
+QUEST_LV_0100_20151001_009544	The grandchild of the elder seems to have some words to tell you after he went out to check whether the Pharets have become more gentle since the collapse of demonic totem. Talk to the grandchild of the elder.
 QUEST_LV_0100_20151001_009545	Talk to Divdirbys Withus
-QUEST_LV_0100_20151001_009546	The grandchild of the elder told you that although the demonic totem has been destroyed, the pharets were still very violent. Talk to Divdirbys, Withus if there's any way to make them calm.
-QUEST_LV_0100_20151001_009547	Divdirbys, Withus was actually enagaged in the research to create a sculpture which has opposite effects as compared to the demonic totem. Talk to Withus.
+QUEST_LV_0100_20151001_009546	The grandchild of the elder told you that although the demonic totem has been destroyed, the Pharets were still very violent. Talk to Divdirbys, Withus if there's any way to make them calm.
+QUEST_LV_0100_20151001_009547	Divdirbys, Withus was actually enagaged in research to create a sculpture which would have an opposite effect as compared to the demonic totem. Talk to Withus.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: Withus was actually engaged in research to create a sculpture which would oppose the effects of the demonic totem.

**Comment by seejy** [comment]
How on earth did i replace a spelling error with another spelling error  _facepalm_

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -9217,16 +9217,16 @@ QUEST_LV_0100_20151001_009212	Go meet Jurate to report
 QUEST_LV_0100_20151001_009213	You've succeeded excavating the divine object using the detector. Bring the divine object to Jurate.
 QUEST_LV_0100_20151001_009214	Restore the records of the merchants
 QUEST_LV_0100_20151001_009215	Talk to Margelious
-QUEST_LV_0100_20151001_009216	Meet Margelious of Kedorlaomer. He seems that he need some help.
+QUEST_LV_0100_20151001_009216	Meet Margelious of Kedorlaomer. He seems that he needs some help.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:9220]: Margelious -> Marcelijus

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -9217,16 +9217,16 @@ QUEST_LV_0100_20151001_009212	Go meet Jurate to report
...
 QUEST_LV_0100_20151001_009217	Bring the records of the merchants
 QUEST_LV_0100_20151001_009218	Please bring back the records which the monsters took away.
-QUEST_LV_0100_20151001_009219	You've found all the records. Bring them to Margelious.
+QUEST_LV_0100_20151001_009219	You've found all of the records. Bring them to Margelious.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:9223]: Margelious -> Marcelijus

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -9217,16 +9217,16 @@ QUEST_LV_0100_20151001_009212	Go meet Jurate to report
...
-QUEST_LV_0100_20151001_009223	In order to restore the records of the merchants, you would need a special dye. To make that dye, you need some fresh body fluid of the monsters. When you put the traps near the monsters, the monsters will get caught so obtain the body fluid at then. Remember the monsters would die quickly when they get caught by the traps so you should pull them out fast. 
+QUEST_LV_0100_20151001_009222	Obtain the body fluid from the monsters
+QUEST_LV_0100_20151001_009223	In order to restore the records of the merchants, you would need a special dye. To make that dye, you need some fresh body fluid from the monsters. When you put the traps near the monsters, the monsters will get caught allowing you to obtain the body fluid. Remember the monsters will die quickly when they get caught by the traps so you should pull them out fast. 
 QUEST_LV_0100_20151001_009224	Bring them to Margelious
-QUEST_LV_0100_20151001_009225	You've obtained all body fluid of the monsters. Bring them to Margelious.
+QUEST_LV_0100_20151001_009225	You've obtained all of the body fluids from the monsters. Bring them to Margelious.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:9229]: Margelious -> Marcelijus

**Pull request closed without merging**